dashboard/app.py

```diff
@@ -211,7 +226,14 @@ def signout():
 @oidc.oidc_auth
 def dashboard():
     """Primary dashboard the users will interact with."""
-    logger.info("User authenticated proceeding to dashboard.")
+    logger.info("User: {} authenticated proceeding to dashboard.".format(session.get('id_token')['sub']))
+
+    if "Mozilla-LDAP" in session.get('userinfo')['sub']:
```

Comment:
note: its Mozilla-LDAP-Dev on dev, though i guess we only use prod atm in which case that'll work

Reply:
Shouldn't it work either way?  It's substring...

Reply:
oh yeah i read it as an array out of habit i guess